BEMT_Hover.py

```diff
@@ -0,0 +1,136 @@
+
+        lambda_i[j] = (-C1 + np.sqrt(C1**2 - 4 * C2)) / 2  # Induced inflow ratio
+
+        phi[j] = mu / r_bar[j] + lambda_i[j] / r_bar[j]  # Inflow angle (rad)
+        alpha[j] = theta[j] - phi[j]  # Angle of attack (rad)
+
+        F_prandtl[j] = 2 /np.pi * np.arccos(np.exp(N_mr / (2 * lambda_i[j]) * (r_bar[j] - 1)))  # Prandtl correction
+                                                                                                 # function
+        if r_bar[j] == 1:
+            F_prandtl[j] = 0
+
+        Cl[j] = Cla * alpha[j]  # Lift coefficient
+        Cd[j] = Cd0  # Drag coefficient
+
+
+        dTcdr_bar[j] = 0.5 * sol * Cl[j] * r_bar[j]**2  # Incremental Thrust coefficient along the main rotor
+
+        dTcdr_bar_PR[j] = dTcdr_bar[j] * F_prandtl[j]   # Incremental Thrust coefficient along the main rotor
+                                                        # considering the Prandtl correction function
+
+        print(dTcdr_bar_PR[j])
+        dQcdr_bar[j] = 0.5 * sol * Cd[j] * r_bar[j]**3 + lambda_i[j] * dTcdr_bar[j]  # Incremental Torque coefficient
+                                                                                     # along the main rotor
+
+        dQcdr_bar_PR[j] = 0.5 * sol * Cd[j] * r_bar[j]**3 + lambda_i[j] * dTcdr_bar_PR[j]  # Incremental Torque
+                                                                                           # coefficient along the main
+                                                                                           # rotor considering the
+                                                                                           # Prandtl correction function
+
+        dPc_i[j] = 0.5 * sol * Cl[j] * phi[j] * r_bar[j]**3  # Incremental induced Power coefficient of the rotor
+                                                             # element
+        dPc_0[j] = 0.5 * sol * Cd[j] * r_bar[j]**3  # Incremental parasite Power coefficient of the rotor element
+
+
+# THRUST, TORQUE, POWER COEFFICIENTS AND FIGURE OF MERIT
+
+    Tc = np.round(np.trapz(dTcdr_bar, r_bar), 4) # Thrust coefficient
+    Tc_PR = np.round(np.trapz(dTcdr_bar_PR, r_bar),4)  # Thrust coefficient  considering the Prandtl correction
+                                                               # function
+
+    Qc = np.round(np.trapz(dQcdr_bar, r_bar),8)  # Torque (Power) coefficient
+    Qc_PR = np.round(np.trapz(dQcdr_bar_PR, r_bar),8)  # Torque (Power) coefficient considering the Prandtl
+                                                               # correction function
+
+    Pc_i = np.round(np.trapz(dPc_i, r_bar),8) # Induced Power (Torque) coefficient
+    Pc_0 = np.round(np.trapz(dPc_0, r_bar),8) # Parasite Power (Torque) coefficient
+
+    FM = ((Tc**(3/2) / np.sqrt(2)))/Qc # Figure of Merit
+    FM_PR = ((Tc_PR ** (3 / 2) / np.sqrt(2))) / Qc_PR  # Figure of Merit considering the Prandtl correction function
```

Comment:
You can avoid computing `Tc`, `Qc` and `FM` with Prandtl's correction.
Just provide in output the uncorrected distributions and the `F_prandtl`.